src/implementations/config/types.ts

```diff
@@ -22,8 +21,19 @@ export type Form = {
   fileName?: string;
 };
 
+export enum NetworkName {
+  local = "local",
+  mainnet = "homestead",
+  goerli = "goerli",
+  sepolia = "sepolia",
+  polygon = "matic",
+  mumbai = "maticmum",
+}
+
+export type Network = "homestead" | "local" | "goerli" | "sepolia" | "matic" | "maticmum";
+
 export interface ConfigFile {
-  network: keyof typeof supportedNetwork;
+  network: Network;
```

Comment:
wah try to make this work instead of defining ur easier custom types

Reply:
ok